x/sidechain/config.go

```diff
@@ -24,3 +24,15 @@ func newCrossChainCfg() *crossChainConfig {
 	}
 	return config
 }
+
+func (c *crossChainConfig) DestChainNameToID(name string) sdk.ChainID {
+	return c.destChainNameToID[name]
+}
+
+func (c *crossChainConfig) ChannelIDs() []sdk.ChannelID {
```

Comment:
you are iterate a map, quite dangerous!

Reply:
ok, I put IDs in the `init` function, all the channels would register at the app starting